docs/DeveloperGuide_Winery.md

```diff
@@ -0,0 +1,622 @@
+---
+{}
+---
+---
+---
+<center>
+
+![alt text](graphics/Figure00-Cloudcycle.png "Cloudcyle")
+
+###Bereitstellung, Verwaltung und vermarktung von portablen Cloud-Diensten mit  
```

Comment:
Use `<bold>`, not markdown headings. Markdown headings have different semantics.